Fix flaky test

The test `test_news_listing_block_filters_by_path` is flaky #108 

```
Failure in test test_news_listing_block_filters_by_path (ftw.news.tests.test_news_listing_block.TestNewsListingBlockContentType)
Traceback (most recent call last):
  File "/var/lib/jenkins/zope/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 340, in run
    testMethod()
  File "/var/lib/jenkins/zope/eggs/ftw.testbrowser-1.30.1-py2.7.egg/ftw/testbrowser/__init__.py", line 42, in test_function
    return func(self, *args, **kwargs)
  File "/var/lib/jenkins/workspace/ftw.news@2/ftw/news/tests/test_news_listing_block.py", line 392, in test_news_listing_block_filters_by_path
    browser.css('.news-item .title').text
  File "/var/lib/jenkins/zope/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 521, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/var/lib/jenkins/zope/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 752, in assertListEqual
    self.assertSequenceEqual(list1, list2, msg, seq_type=list)
  File "/var/lib/jenkins/zope/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 734, in assertSequenceEqual
    self.fail(msg)
  File "/var/lib/jenkins/zope/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 415, in fail
    raise self.failureException(msg)
AssertionError: Lists differ: ['A News Item in News Folder 1... != ['A News Item in News Folder 2...

First differing element 0:
A News Item in News Folder 1
A News Item in News Folder 2

- ['A News Item in News Folder 1', 'A News Item in News Folder 2']
?                              ^                               ^

+ ['A News Item in News Folder 2', 'A News Item in News Folder 1']
?                              ^                               ^
```

--- a/docs/HISTORY.txt
+++ b/docs/HISTORY.txt
@@ -5,7 +5,7 @@ Changelog
 1.11.2 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Bugfix: Adapt flaky test to not check list order [busykoala]
 
 
 1.11.1 (2019-03-22)

--- a/ftw/news/tests/test_news_listing_block.py
+++ b/ftw/news/tests/test_news_listing_block.py
@@ -384,7 +384,7 @@ class TestNewsListingBlockContentType(FunctionalTestCase):
         # By default, the block renders news from the children of
         # its container.
         browser.visit(block.absolute_url() + '/block_view')
-        self.assertEqual(
+        self.assertItemsEqual(
             [
                 'A News Item in News Folder 1',
                 'A News Item in News Folder 2',

